test/mailers/notifier_test.rb

```diff
@@ -1,10 +1,18 @@
 require 'test_helper'
 
 class NotifierTest < ActionMailer::TestCase
+  fixtures :lists
 
   def setup
-    recipients = User.all.pluck :email
-    options = {recipients: recipients, completed: 0, remaining: 5}
+    list = lists(:first)
+
+    options = {
+      recipients: ['vasyll@tada.com'],
+      completed_todos: {list => list.items},
```

Comment:
Can you use `{list: list.items}` notation here instead of rocked hash. Please check other places.